var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -89,4 +106,58 @@ def configure_args(self):
                 '--with-libint-max-am=5',
                 '--with-libderiv-max-am1=4'
             ])
+
+        if '@2.6.0:' in self.spec:
+            config_args += ['--with-libint-exportdir=generated']
+
+            tune_value = self.spec.variants['tune'].value
+            if tune_value.startswith('cp2k'):
+                lmax = int(tune_value.split('-lmax-')[1])
+                config_args += [
+                    '--enable-eri=1',
+                    '--enable-eri2=1',
+                    '--enable-eri3=1',
+                    '--with-max-am={0}'.format(lmax),
+                    '--with-eri-max-am={0},{1}'.format(lmax, lmax - 1),
+                    '--with-eri2-max-am={0},{1}'.format(lmax + 2, lmax + 1),
+                    '--with-eri3-max-am={0},{1}'.format(lmax + 2, lmax + 1),
+                    '--with-opt-am=3',
+                    '--enable-generic-code',
+                    '--disable-unrolling',
+                ]
+
         return config_args
+
+    @property
+    def build_targets(self):
+        if '@2.6.0:' in self.spec:
+            return ['export']
+
+        return []
+
+    @when('@2.6.0:')
+    def install(self, spec, prefix):
+        """
+        Starting from libint 2.6.0 we're using the 2-stage build
+        to get support for the Fortran bindings, required by some
+        packages (CP2K notably).
+        """
+
+        # upstream says that using configure/make for the generated code
+        # is deprecated and one should use CMake, but with the currently
+        # recent 2.7.0.b1 it still doesn't work
```

Comment:
@adamjstewart @tgamblin Moving the build system of a package from `Makefile` or `autotools` to `CMake` happens frequently lately. Maybe the community should discuss a common approach to do this moving. For the time being I think this code is fine.

Reply:
Here we would move from a configure+configure to a configure+cmake, which is even more messed up.
The way Linux (meta-)distributions are doing it is to have one build script per version. For binary distros are the old versions kept in a VCS, for source distros (like Gentoo) they're kept as separate scripts (resp. copies of other scripts). This allows to follow build system changes of upstream projects in version bumps without interfering with already tested/working build scripts for older versions of the package.